applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/stress_response_definitions.h

```diff
@@ -127,6 +128,11 @@ class StressCalculation
                             Vector& rOutput,
                             const ProcessInfo& rCurrentProcessInfo);
 
+    static void CalculateStressOnGPSmallDisplacement(Element& rElement,
```

Comment:
better use a namespace in the future

```suggestion
    static void CalculateStressOnGPSmallDisplacement(const Element& rElement,
```

Reply:
That does not work due to the internal call of  "CalculateOnIntegrationPoints" from the element.